Makefile

```diff
@@ -82,7 +82,7 @@ pd-server-basic:
 pd-ctl:
 	CGO_ENABLED=0 go build -gcflags '$(GCFLAGS)' -ldflags '$(LDFLAGS)' -o $(BUILD_BIN_PATH)/pd-ctl tools/pd-ctl/main.go
 pd-tso-bench:
-	CGO_ENABLED=0 go build -o $(BUILD_BIN_PATH)/pd-tso-bench tools/pd-tso-bench/main.go
+	cd tools/pd-tso-bench && CGO_ENABLED=0 go build -o $(BUILD_BIN_PATH)/pd-tso-bench main.go
```

Comment:
BTW, why do we need to change it?

Reply:
we need to cd to the directory to use the `go.mod` of submodule.

Reply:
Oh, I get it.